### ISSUE TITLE
inconsistent case of PCbuild/ directory

### ISSUE THREAD

**Issue by 5d22bdbc-bd0d-4723-8cd4-68e20aca459e** [python/cpython#75750]
BPO | [31569](https://bugs.python.org/issue31569)
--- | :---
Nosy | @pfmoore, @tjguk, @zware, @serhiy-storchaka, @zooba, @gronke
PRs | <li>python/cpython#3711</li>

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = <Date 2017-09-25.18:36:04.288>
created_at = <Date 2017-09-24.11:44:44.430>
labels = ['type-feature', '3.7', 'OS-windows', 'build', 'docs']
title = 'inconsistent case of PCbuild/ directory'
updated_at = <Date 2017-09-25.18:36:04.287>
user = 'https://github.com/gronke'
```

bugs.python.org fields:
```python
activity = <Date 2017-09-25.18:36:04.287>
actor = 'steve.dower'
assignee = 'docs@python'
closed = True
closed_date = <Date 2017-09-25.18:36:04.288>
closer = 'steve.dower'
components = ['Build', 'Documentation', 'Windows']
creation = <Date 2017-09-24.11:44:44.430>
creator = 'gronke'
dependencies = []
files = []
hgrepos = []
issue_num = 31569
keywords = []
message_count = 7.0
messages = ['302851', '302852', '302857', '302959', '302960', '302961', '302963']
nosy_count = 7.0
nosy_names = ['paul.moore', 'tim.golden', 'docs@python', 'zach.ware', 'serhiy.storchaka', 'steve.dower', 'gronke']
pr_nums = ['3711']
priority = 'normal'
resolution = 'fixed'
stage = 'resolved'
status = 'closed'
superseder = None
type = 'enhancement'
url = 'https://bugs.python.org/issue31569'
versions = ['Python 2.7', 'Python 3.3', 'Python 3.4', 'Python 3.5', 'Python 3.6', 'Python 3.7']
```

</p></details>

**Comment by 5d22bdbc-bd0d-4723-8cd4-68e20aca459e** [comment]
While reading the build documentation for Windows, I've noticed the `PCbuild/` directory to be mentioned with a different case than the directory in the repository (`PCBuild/` instead of `PCbuild/`).

Every occasion of `PCBuild` should be replaced with `PCbuild`, so that builds can work on case-sensitive filesystems as well.

**Comment by serhiy-storchaka** [comment]
Wouldn't be easier to make changes in the opposite direction? Change PCbuild to PCBuild?

The file systems on Windows are case-insensitive, so this hardly an error.

**Comment by pfmoore** [comment]
I would strongly prefer the docs to be changed to match the implementation rather than the other way round. However, I don't see the use case for this - what case sensitive filesystem would anybody be building the Windows version of Python on?


I'm -1 on changing the actual build scripts, as that introduces risk for no actual benefit. I'm +0 on changing the docs, assuming it's possible to make things consistent with a doc-only change. But I can't imagine anyone reading those docs would expect a filesystem to be case sensitive, so I doubt it would matter in practice.

**Comment by zooba** [comment]
I'm okay with the PR as it stands, especially the parts that affect GitHub and appveyor, as those are almost certainly using case sensitive comparisons.

It's definitely not easier to update the case of a file that is in git, and doing so will very likely break users (we had to do a case-folding pass over the Mercurial repository because while it could handle case changes, git could not import the ones we had in history).

Personally I'd prefer it to have always been PCBuild rather than PCbuild, and it's very likely that I've introduced most of the 'incorrect' ones. But what's done is done and we should avoid the potential distraction of mismatched case throughout the rest of our codebase.
